flytekit/exceptions/scopes.py

```diff
@@ -15,6 +16,7 @@
 
 class FlyteScopedException(Exception):
     def __init__(self, context, exc_type, exc_value, exc_tb, top_trim=0, bottom_trim=0, kind=None):
+        warn(f"{self.__class__.__name__} is deprecated.", DeprecationWarning, stacklevel=2)
```

Comment:
When do we remove deprecated items? flytekit 2.0?

Reply:
Don't want to wait until 2.0, maybe 6 months after

Reply:
Can you create a github issue to track this deprecation?